IM150/admin/admin_forum_tour.php

```diff
@@ -163,7 +165,7 @@ function reorder_pages()
 			$message = prepare_message(trim($message), 1, 1, 1, $bbcode_uid);
 
 			$sql = "UPDATE " . FORUM_TOUR_TABLE . "
-				SET page_subject = '$subject', page_text = '$message', page_access = $page_access
+				SET page_subject = '$subject', page_text = '$message', page_access = '$page_access'

```

Comment:
that's just it. I am not good with sql at all. Much like you mentioned the other day you did not want to touch .css. I hate it. I worked with these two inserts and updates until i was blue in the face and adding the ticks is all that made them not error out.

As far as getting the text to print back on the screen in the box in ACP, I could not do it. that one might be beyond what I am capable of.